src/bidiMapper/domains/protocol/parsers/sessionParser.ts

```diff
@@ -0,0 +1,41 @@
+import { Session as SessionType } from '../bidiProtocolTypes';
+import { InvalidArgumentErrorResponse } from '../error';
+import { BrowsingContextParser } from './browsingContextParser';
+
+export namespace SessionParser {
```

Comment:
I think it'd be useful to have unit tests for the parser. Or do you plan to rely on WPT tests for this?

Reply:
done for checking if event or BrowsingContext are strings

Reply:
done in helper